lte/gateway/configs/pipelined.yml_prod

```diff
@@ -128,6 +128,9 @@ redis_enabled: true
 # Logs grpc payload content
 magma_print_grpc_payload: false
 
+# MTR iface
+mtr_interface: mtr0
+
```

Comment:
classifier.py and inout also uses mtr_ip.
Lets move this main.py and read interface IP, so that these APP does not need any changes.